pkg/resources/management.cattle.io/v3/fleetworkspace/validator_test.go

```diff
@@ -0,0 +1,99 @@
+package fleetworkspace
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/webhook/pkg/admission"
+	"github.com/rancher/webhook/pkg/fakes"
+	admissionv1 "k8s.io/api/admission/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func TestAdmit(t *testing.T) {
+	nsName := "test"
+
+	nsExistMutator := func() Mutator {
+		ctrl := gomock.NewController(t)
+		mockNamespaceController := fakes.NewMockNamespaceController(ctrl)
```

Comment:
We no longer need to generate fakes for Wrangler controllers. Instead, we can use the new [generic/fake](https://github.com/rancher/wrangler/tree/master/pkg/generic/fake) package from wrangler.
Hear is an [example](https://github.com/rancher/webhook/pull/262/files#diff-7d2acc95d10395b594e7b75b2516089d511d173afa33235d3a8d0d87de41ea44) of how that change looks.

This uses gomock as well, so you shouldn't need to change any of your test logic.

Reply:
done in https://github.com/rancher/webhook/pull/260/commits/24334874f8922d5187275b7fdf0f1c85dfcacf5f